test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -9204,10 +9204,6 @@ public void ToNDArray()
                 Assert.Equal(2, a.GetLength(5));
                 Assert.Equal(expected, a);
             }
-            {
-                var t = torch.rand(new long[] { 10, 20, 30, 2, 4, 8, 16 });
-                Assert.Throws<NotImplementedException>(() => t.data<float>().ToNDArray());
-            }
```

Comment:
Maybe I should modify this test to test my code

Reply:
Definitely add unit tests.

Reply:
Done.